dbstore.go

```diff
@@ -26,7 +26,7 @@ import (
 // Store ... TODO(pascal): write about abstraction.
 type Store interface {
 	// Load loads the worksheet with identifier `id` from the store.
-	Load(id string) (*Worksheet, error)
+	Load(id string, version ...int) (*Worksheet, error)
```

Comment:
My hunch is that I'd prefer having `LoadAt(id string, version int)` distinct from `Load`:
- Use cases for one versus the other are quite different
- The behavior is quite different, as you describe `Load(id)` and `Load(id, head_version)` produce different results